source/kraken/realtime.cpp

```diff
@@ -120,10 +136,15 @@ static std::ostream& operator<<(std::ostream& s, const nt::StopTime& st) {
  */
 static bool check_disruption(const nt::disruption::Disruption& disruption) {
```

Comment:
Probably not in this PR: we have 2 checks on consistency... Can't we group them?
The same goes on Kirin, and it does not seem easy to do there.

Reply:
Will do in the next PR no?